core/domain/exp_domain.py

```diff
@@ -777,8 +994,29 @@ def from_dict(cls, interaction_dict):
 
     def __init__(
             self, interaction_id, customization_args, answer_groups,
+<<<<<<< HEAD
```

Comment:
Whoops, bad merge. Careful! Make sure that, after you've fixed everything, the diff in the "Files Changed" tab is exactly what you want it to be.